examples/esp32/partitions.csv

```diff
@@ -2,4 +2,4 @@
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
 nvs,      data, nvs,     ,        0x6000,
 phy_init, data, phy,     ,        0x1000,
-factory2,  app,  factory, ,       0x360000,
```

Comment:
I would like to be conscious about the flash usage here since we want to reserve some for future use (see last commit in micro-rdk) Do you have an idea of the current size it takes with your updates?

Reply:
release: `App/part. size:    3,065,872/4,128,768 bytes, 74.26%`
debug: `App/part. size:    3,964,080/4,128,768 bytes, 96.01%`

Reply:
When using the original partitions

```
Partition table:   esp32/partitions.csv
Error: espflash::image_too_big (link)

  × Supplied ELF image of 3964080B is too big, and doesn't fit configured app partition of 3538944B
  help: Reduce the size of the binary or increase the size of the app partition.
```

Reply:
increased partition size slightly over original to fit code with webhook implementation.